src/scenes/characters/character.gd

```diff
@@ -33,6 +33,21 @@ func check_quest_status() -> void:
 
 func dialog_ended(_timeline_name) -> void:
```

Comment:
Now that you are consuming the parameter value, you should remove the leading underscore.

Reply:
Good point -- this has been updated.